Minor improvements in Serverless docs (#9064)
kyma-project/kyma: docs/serverless/05-02-serverless-chart-config.md

This table lists the configurable parameters, their descriptions, and default values for both cluster and local installations.

> > **NOTE:** Limited memory and CPU resources on Minikube directly affects the Serverless functionality as you can process only a limited number of Functions at the same time, and their processing time is significantly longer. To process large workloads, we recommend using the cluster setup.

Reviewer: ```suggestion
> **NOTE:** Limited memory and CPU resources on Minikube directly affect the Serverless functionality as you can process only a limited number of Functions at the same time and their processing time is significantly longer. To process large workloads, we recommend using the cluster setup.
```

Reviewer: And to shorten this sentence, you could split it as: 
`Limited memory and CPU resources on Minikube directly affect the Serverless functionality as you can process only a limited number of Functions at the same time. Also, their processing time is significantly longer.`
or
`Limited memory and CPU resources on Minikube directly affect the Serverless functionality as you can process only a limited number of Functions at the same time. Their processing time is significantly longer too.` or `as well.`

Again, I leave it to your discretion.

| Parameter                                       | Description                                                           | Default value | Minikube override |
| ----------------------------------------------- | --------------------------------------------------------------------- | ------------- | ----------------- |
| **containers.manager.envs.buildRequestsCPU**    | Number of CPUs requested by the image-building Pod to operate.        | `700m`        | `100m`            |
| **containers.manager.envs.buildRequestsMemory** | Amount of memory requested by the image-building Pod to operate.      | `700Mi`       | `200Mi`           |
| **containers.manager.envs.buildLimitsCPU**      | Maximum number of CPUs available for the image-building Pod to use.   | `1100m`       | `200m`            |
| **containers.manager.envs.buildLimitsMemory**   | Maximum amount of memory available for the image-building Pod to use. | `1100Mi`      | `400Mi`           |

> > **TIP:** See the [official documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes) for details on resource units in Kubernetes.

Reviewer: I'm afraid `official documentation` might not be any more descriptive for SEO than `here`.
```suggestion
> **TIP:** To learn more, read the official documentation on [resource units in Kubernetes](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes).
```
